package.json

```diff
@@ -1,22 +1,27 @@
 {
   "name": "carbon-addons-iot-react",
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
+  "module": "es/src/index.js",
+  "files": [
+    "lib/**/*",
+    "es/**/*",
+    "umd/**/*",
+    "scss/**/*"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/IBM/carbon-addons-iot-react.git"
   },
   "license": "Apache-2",
-  "files": [
-    "lib"
-  ],
   "bugs": {
     "url": "https://github.com/IBM/carbon-addons-iot-react/issues"
   },
   "homepage": "https://carbon-addons-iot-react.com",
   "scripts": {
-    "build": "yarn build:pre && rollup -c",
-    "build:pre": "rimraf lib storybook-static",
+    "build": "yarn build:pre && yarn build:main",
+    "build:pre": "rimraf lib es umd scss storybook-static",
     "build:storybook": "NODE_OPTIONS='--max-old-space-size=16384' build-storybook -s public/production",
+    "build:main": "rollup -c ",
```

Comment:
Minor nit: move `build:main` up two lines to keep them alphabetical. 
```json
    "build": "yarn build:pre && yarn build:main",
    "build:main": "rollup -c ",
    "build:pre": "rimraf lib es umd scss storybook-static",
    "build:storybook": "NODE_OPTIONS='--max-old-space-size=16384' build-storybook -s public/production",
```

Reply:
using same commands since using rollup for all versions.